### PR TITLE
Enable modifying logout button text and class in googleSignInUI and expose sign in status in googleSignIn

### PR DESCRIPTION
**Pull request by romunov**
This PR enables the logout button to be customizable by specifying its name and class. It also exposes the signed in status via `googleSignIn` module.

Also, let me know how you score versions.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: googleAuthR
 Type: Package
-Version: 0.8.1.9000
+Version: 0.8.2.9000
```
Comment (MarkEdmondson1234) [inline DESCRIPTION]: This would be `0.8.1.9001` - `9000` is the dev versioning not on CRAN, once on CRAN it would be `0.8.2`

**Comment by romunov** [comment]
Thanks. I'll also peruse through the docs again, I think I saw an inconsistency in there somewhere...

File: DESCRIPTION (hunk 1 above)
Comment (romunov) [reply DESCRIPTION]: Thanks. I'll also peruse through the docs again, I think I saw an inconsistency in there somewhere...

**Comment by MarkEdmondson1234** [comment]
Thanks!

**Pull request merged**